racket/collects/syntax/to-string.rkt

```diff
@@ -58,7 +58,19 @@
                (define c-paren-shape (syntax-property c 'paren-shape))
                (printf "~a" (or c-paren-shape #\())
                (set! col (+ col 1))
-               (map (loop init-line!) (syntax->list c))
+               (letrec ([se (syntax-e c)]
+                        [build-string-from-pair (lambda (sp)
+                                                  (cond
+                                                    [(syntax? sp)
+                                                      (printf " . ")
```

Comment:
Nit: wrong indentation. DrRacket's auto indentation would align the left paren with the left paren on the above line.